code/modules/mining/ores_coins.dm

```diff
@@ -60,6 +61,20 @@
 	qdel(src)
 	return
 
+/obj/item/weapon/ore/glass/throw_impact(atom/hit_atom)
+	if(..() || !iscarbon(hit_atom))
+		return
+	var/mob/living/carbon/C = hit_atom
+	if((C.head && C.head.flags_cover & HEADCOVERSEYES) || \
+		(C.wear_mask && C.wear_mask.flags_cover & MASKCOVERSEYES))
+		visible_message("<span class='danger'>[C]'s mask/helmet blocks the sand!</span>")//glasses are not thick enough to stop it

```

Comment:
You would need to insert the name of the mask or helmet as a [var] to get it to use the correct name; 'make/helmet' does nothing